definitions.yaml

```diff
@@ -55,6 +55,27 @@ definitions:
               default:
                 description: The instance type used in new cluster, if not specified
                 type: string
+          volumes:
+            description: Information related to volumes on worker nodes
+            type: object
+            properties:
+              docker:
+                description: Information about the docker volume attached to the worker node. This volume is used to store docker images.
```

Comment:
> This volume is used to store docker images.

Is this correct/complete? I reckon that running container's overlay file systems are also placed there.

Reply:
resolved